alma/views/templates/hook/payment_button_desc.tpl

```diff
@@ -24,3 +24,12 @@
 <p>
     {$desc|escape:'htmlall':'UTF-8'}
 </p>
+{foreach from=$plans item=v name=counter}        
+        
+        <p class="alma-p-fee-plan--descritpion">    
+            <span>Echéance {$smarty.foreach.counter.iteration} : le {$v.due_date|date_format:"%d/%m/%Y"}</span>
+            <span>{math equation=$v.purchase_amount / 100 format="%.2f"}&euro;</span>
+        </p>
+         
+    {/foreach}
+    <br />
```

Comment:
identation issues?

Reply:
+ same display changes as in displayPayment.tpl

👉 could we refactor the installments display in a single tpl file, included by both "master" templates?